src/api/app/models/review.rb

```diff
@@ -25,6 +27,38 @@ def state
     read_attribute(:state).to_sym
   end
 
+  def check_initial
+    # Validates the existens of references.
+    # NOTE: they can disappear later and the review should be still
+    #       usable to some degree (can be showed at least)
+    #       But it must not be possible to create one with broken references
+    unless self.by_user or self.by_group or self.by_project
+      errors.add(:unknown, 'no reviewer defined')
+    end
+
+    if self.by_user and not User.find_by_login(self.by_user)
```

Comment:
yes, but I wanted to use the find_by_login method in any case, since it does extra stuff like checking the user state.